gradle/dependencies.gradle

```diff
@@ -6,12 +6,17 @@ ext {
       compileSdkVersion       : 29
   ]
 
+  def mapboxNavigatorVersion = System.getenv("FORCE_MAPBOX_NAVIGATION_NATIVE_VERSION")
```

Comment:
For posterity, could you add an inline comment to describe what this is used for?

Reply:
Done.